examples/prediction/HousingServe.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, model_file=MODEL_FILE):
   def predict(self, X, feature_names):
     """Predict using the model for given ndarray."""
     prediction = self.model.predict(data=X)
-    return [[prediction.item(0), prediction.item(0)]]
+    return [[prediction.item(0), prediction.item(1)]]
```

Comment:
@karthikv2k Is this change correct? Its possible that the prediction.item(0) was intentional as the prediction is returning a singleton, but the API requires a calls and confidence?

Reply:
should be returning all the results for the input X - something like return [prediction]

Reply:
Got it, thanks!